lib/src/IndexingPolicy.cpp

```diff
@@ -37,15 +37,17 @@ using namespace web::json;
 IndexingPolicy::IndexingPolicy()
 	: automatic_(true)
 	, indexing_mode_(IndexingMode::CONSISTENT)
-	, excluded_paths_(0)
+	, excluded_paths_()
 	, included_paths_(0)
 {
+	included_paths_.push_back(make_shared<Index>(IndexType::HASH, 3, 3, _XPLATSTR("/")));

```

Comment:
Can you point me where did you see these paths are default paths, we need at least comment in the code about this?